content/200-orm/200-prisma-client/100-queries/090-raw-database-access/050-raw-queries.mdx

```diff
@@ -575,6 +713,44 @@ const result = await prisma.$queryRawUnsafe(query)
 console.log(result)
 ```
 
+#### Parameterized queries
```

Comment:
I struggle a bit with putting the whole "Parametrized queries" section under "SQL Injection". Aren't all the examples shown here safe, and thus it should potentially be listed above where it was before? Why should this be in the "this is all scary and bad" corner of the documentation page?

Reply:
I personally think it makes sense here, especially if we rebrand the section to "SQL injection prevention". The primary use for parameterization in dynamic queries seems to be SQL injection prevention anyway and this way all the guidance around this method is together.

Reply:
Ok, I can get behind that I think. I'll reread the page after the merge, and then see if I am still missing that information above with `$queryRawUnsafe()`.